asciify.py

```diff
@@ -82,5 +82,11 @@ def runner(path):
 '''
 if __name__ == '__main__':
     import sys
+    import urllib.request
+    if sys.argv[1].startswith('http://') or sys.argv[1].startswith('https://'):
+        urllib.request.urlretrieve(sys.argv[1], "asciify.jpg")
+        path = "asciify.jpg"
+    else:
+        path = sys.argv[1]
     path = sys.argv[1]

```

Comment:
Sorry, somehow missed this. Need to delete this line.

Reply:
No worries. It's taken care of. :slightly_smiling_face: